crates/utils/src/sp1.rs

```diff
@@ -9,12 +9,6 @@ mod actual_impl {
     /// Can be any number, as long as it doesn't conflict with default/other hooks.
     pub const FD_METRICS_HOOK: u32 = 1001;
 
-    pub fn cycle_count_hook(env: sp1_sdk::HookEnv, _buf: &[u8]) -> Vec<Vec<u8>> {
-        vec![Vec::from(
-            env.runtime.report.total_instruction_count().to_le_bytes(),
-        )]
-    }
```

Comment:
Just checking if it is intentional to remove that method

Reply:
If so, then feel free to merge

Reply:
Yep, this is getting moved into the adapter. Thanks!